dev/tools/test-migration.py

```
@@ -467,6 +478,36 @@ def upgrade_and_migrate(method, image_prefix, image_tag):
             upgrade_cmd.extend(["--set", f"image.tag={image_tag}"])
         run_cmd(upgrade_cmd)
 
+    # Re-apply the owner reference patch and status binding to protect against the old controller having overwritten it during Phase 2
+    print("Re-applying ownerReference and status patch to secure warm claim binding...")
+    claim_uid = ""
+    for i in range(10):
```

Ruff (0.15.18)

[warning] Loop control variable `i` not used within loop body

Rename unused `i` to `_i`

(B007)